src/components/WalletSelector/multichain/tezos/tezosUtils.ts

```diff
@@ -19,3 +20,9 @@ export function getNonceNumber(nonce: string) {
   const splittedNonceMessage = nonce.split(' ');
   return splittedNonceMessage[splittedNonceMessage.length - 1];
 }
+
+export function getErrorCode(error: BeaconError) {
+  // https://typedocs.walletbeacon.io/enums/beaconerrortype.html
+  // Example: [ABORTED_ERROR]:The action was aborted by the user.
+  return error.message.split(':')[0].replace('[', '').replace(']', '');
```

Comment:
The beacon error didnt provide any `code` properties in their response just like ethereum. 

Need to trim their error message to get their unique code 🤦🏻‍♂️

Reply:
hehehe